modules/statics/src/coins.ts

```diff
@@ -1252,4 +1254,136 @@ export const coins = CoinMap.fromCoins([
     '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e',
     UnderlyingAsset['avaxc:USDC']
   ),
+  solToken(
+    'sol:SRM',
+    'Serum',
+    6,
+    'SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt', // https://explorer.solana.com/address/SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt
+    'SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt',
+    UnderlyingAsset.SRM,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'SRM'
+  ),
+  solToken(
+    'sol:ORCA',
+    'Orca',
+    6,
+    'orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE', // https://explorer.solana.com/address/orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE
+    '8DzsCSvbvBDYxGB4ytNF698zi6Dyo9dUBVRNjZQFHSUt',
```

Comment:
this is the mint authority, which shouldn't be needed in token metadata.

also, this authority can potentially change (i.e. owner of the mint decides to change authority to a different account for whatever reason).